akka-docs/src/main/paradox/stream/operators/Source/unfoldResource.md

```diff
@@ -16,12 +16,40 @@ Wrap any resource that can be opened, queried for next element (in a blocking wa
 
 Wrap any resource that can be opened, queried for next element (in a blocking way) and closed using three distinct functions into a source.
 
+The functions are by default called on the blocking io dispatcher to avoid interfering with other stream operations. 
+See @ref:[Blocking Needs Careful Management](../../../typed/dispatchers.md#blocking-needs-careful-management) for an explanation on why this is important
 
 @@@div { .callout }
```

Comment:
in some of these we have placed this at the end in a separate section `## Reactive Streams semantics`, for example in collect.md